src/midifallback.c

```diff
@@ -111,6 +112,11 @@ static boolean GetProgramFallback(byte idx, byte program,
             // Fall to capital when no instrument has (successfully)
             // selected this variation or if the variation is above 63.
             fallback->value = 0;
+
+            fprintf(stderr,
```

Comment:
How much spam can these generate? Should they be limited to once only? Or once per .midi?

Reply:
See above.